pinot-common/src/main/java/org/apache/pinot/common/function/FunctionUtils.java

```diff
@@ -56,6 +61,12 @@ private FunctionUtils() {
     put(Double.class, PinotDataType.DOUBLE);
     put(String.class, PinotDataType.STRING);
     put(byte[].class, PinotDataType.BYTES);
+    put(int[].class, PinotDataType.INTEGER_ARRAY);
+    put(int.class, PinotDataType.INTEGER);
```

Comment:
Why do you need `int` here? The argument should always be `Object`

Reply:
int was not added for SV, so added int.